pkg/metrics/reporter.go

```diff
@@ -21,6 +21,10 @@ import (
 var clusterIDMut sync.Mutex
 var clusterID *uuid.UUID
 
+func metricsEnabled() bool {
```

Comment:
why not (re-)use the `DisableMetrics` function in `pkg/metrics/airgap.go` that we already use for airgap? can have it respect this env var for example.